docker-compose.yml

```diff
@@ -58,12 +58,13 @@ services:
             - TOKEN_KEY=${SECRET_KEY_JWT}
             - WEB_SERVER_URL=django
             - WEB_SERVER_PORT=8000
-            - QUORIDOR_HOST_PORT=quoridor:50051
+            - WUMPUS_HOST_PORT=wumpus:50051
             - REDIS_HOST=redis
```

Comment:
so it will not be possible to play quoridor? i mean the idea is that both game should be available

Reply:
At this moment. The idea of this sprint is to play only wumpus with actual infrastructure. When the game is tested and working. In future sprints we have the idea to implement a change where you can select between different games